var/spack/repos/builtin/packages/libsignal-protocol-c/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class LibsignalProtocolC(CMakePackage):
+    """C library for the Signal protocol"""
+
+    homepage = "https://signal.org/en/"
+    url      = "https://github.com/signalapp/libsignal-protocol-c/archive/refs/tags/v2.3.3.tar.gz"
+
+    maintainers = ['pwablito']
+
+    version('2.3.3', sha256='c22e7690546e24d46210ca92dd808f17c3102e1344cd2f9a370136a96d22319d')
+
+    def cmake_args(self):
+        args = []
+        return args
```

Comment:
You can remove this if you're not going to add any arguments since the parent class (i.e., `CmakePackage`) already has `return []`.

```suggestion
```